src/language-graphql/printer-graphql.js

```diff
@@ -669,7 +669,7 @@ function determineInterfaceSeparator(originalSource) {
   if (end === -1) {
     end = originalSource.length;
   }
-  return originalSource.substr(start, end).includes("&") ? " & " : ", ";
+  return originalSource.slice(start, end).includes("&") ? " & " : ", ";
```

Comment:
And this. Please double check other changes.

Reply:
fixed in #7305